### ISSUE TITLE
Prefix crawling is crawling complete bucket instead of specific folder

### ISSUE THREAD

**Issue by mvidhu** [data-dot-all/dataall#428]
### Describe the bug

When we enable prefix in the start crawl section and provide specific folder name to crawl, still the crawler is creating many tables instead of only specific table within the prefix.

Eg: I want to crawl specific folder sample inside my bucket, I am specifying the prefix as sample/. I also tried multiple combinations like /sample, sample, /sample/. All the combinations did not work and I always have crawler create tables for entire bucket structure.

### How to Reproduce

```
*P.S. Please do not attach files as it's considered a security risk. Add code snippets directly in the message body as much as possible.*
```
Import a S3 bucket containing many folders to data.all
Start a crawler with prefix toggle enabled and provide a folder name for prefix.
Refresh after few minutes, either the crawler creates all tables or sometimes none.

### OS

Mac

### Python version

3.11

### AWS data.all version

0.5.0

**Comment by dlpzx** [comment]
@mvidhu Thanks for raising the issue, we are looking into it :)

**Comment by dlpzx** [comment]
After May 8th we will pick it up

**Comment by dlpzx** [comment]
@mvidhu this item will be released in v2.4.0
